src/detect-engine-event.c

```diff
@@ -160,6 +160,9 @@ static DetectEngineEventData *DetectEngineEventParse (const char *rawstr)
 
     if (de->event == STREAM_REASSEMBLY_OVERLAP_DIFFERENT_DATA) {
         StreamTcpReassembleConfigEnableOverlapCheck();
+    } else if (de->event == UDP_HLEN_INVALID) {
```

Comment:
I think this should be implemented using the strict concept as done here: https://github.com/OISF/suricata/blob/master/src/detect-app-layer-event.c#L201

cc @lukashino

Reply:
@jufajardini this strict concept should make it to the dev guide :)

Reply:
followed the concept and created `OutdatedEvent` function in decode part of the engine.